go/apps/http_api/tests/test_vumi_app.py

```diff
@@ -482,7 +482,8 @@ def test_invalid_in_reply_to_with_missing_conversation_key(self):
         # create a message with no conversation
         inbound_msg = self.app_helper.make_inbound('in 1', message_id='msg-1')
         vumi_api = self.app_helper.vumi_helper.get_vumi_api()
-        yield vumi_api.mdb.add_inbound_message(inbound_msg)
+        opms = vumi_api.get_operational_message_store()
```

Comment:
The new message store is split into three pieces:
* The batch manager does stuff related to batches.
* The operational message store reads and writes messages by key only and is used by middleware and routing code.
* The query message store is read-only and is used by everything that needs to list, search, filter, etc.